### DISCUSSION TITLE
More emojis with teeth

### DISCUSSION THREAD

**Discussion by ghost** [C1710/blobmoji#88] [Style]
Update these emojis right here: 😄😆😅🤣 so those emojis have teeth. The teeth less ones can be in the extension pack combining 😄+😶 and any of the other emojis that I said in this comment, so those ZWJ-sequences could have no teeth.

**Comment by ghost** [comment]
Did you update them yet?

**Reply by C1710** [reply]
No and I probably won't, since teeth don't look really good on these emojis there and are not required to understand the emojis.

**Reply by C1710** [reply]
The Face with Tears of Joy even had teeth, but I removed them.
